### ISSUE TITLE
Tint example not working

### ISSUE THREAD

**Issue by MartinSeeler** [JonasKruckenberg/imagetools#301]
In your directive examples, it seems like the `tint` option is not working in vite-imagetool.

The example says
```jsx
import Image from 'example.jpg?tint=#ffaa22'
import Image from 'example.jpg?tint=rgba(10,33,127)'
```

But it looks like vite-imagetools prefixes the expected color with a `#` anyway. The result is, that when using a rgb color, it fails with

```
Unable to parse color from string: #rgba(10,33,127)
```

also the hex example does fail with

```
Unable to parse color from string: ##ffaa22
```

It works when no leading `#` is provided, but rgb is impossible to achieve.

**Comment by JonasKruckenberg** [comment]
Yes indeed, the argument passed to `tint` is prefixed with a `#`. And there was a reason for it, I just don't remember why exactly.
